README.md

```diff
@@ -200,26 +200,38 @@ connect to your VMs and figure things out.
 terraform output -state <your_state_file>.tfstate instancePublicIPs
 ```
 
-2. Export a private key to the file:
+2. Remove outadeted private key (if exist)
+
+```shell
+rm -f <regionName>.pem
+```
+
+3.  Export a private key to the file:
 
 ```shell
 terraform output -raw -state <your_state_file>.tfstate tls_private_key > <regionName>.pem
 ```
 
-3. Add a read access to the key file:
+4. Add a read access to the key file:
 
 ```shell
 chmod 400 <regionName>.pem
 ```
 
-4. Connect via SSH:
+5. Connect via SSH:
 
 ```shell
 ssh -i <regionName>.pem azureuser@<your_vm_IP>
 ```
 
-5. Check a status of the `mhddos` docker container:
+6. Check a status of the `mhddos` docker container:
 
 ```shell
 sudo docker ps
 ```
+
+7. Attach to container terminal
+
+```shell
+sudo docker attach <container name/id>
```

Comment:
Replace <container name/id> with mhddos

Reply:
Replaced with <mhddos/mhddosProxy> for two ways of ddos command.